releases/v1.5.0-beta.toml

```diff
@@ -48,6 +48,10 @@ brings support for the Node Resource Interface (NRI).
 ## Windows
 * **Optimize LCOW snapshotter use of scratch layers** [#4643](https://github.com/containerd/containerd/pull/4643)
 
+## CRI
+* **Update privileged containers to use current capabilities instead of known capabilities** [#5017](https://github.com/containerd/containerd/pull/5017)
```

Comment:
Not specific to CRI

Reply:
What would you suggest here? Trying to limit the categorization in the release notes since we don't normally include library level changes or `ctr`. The most user impactful part here I think is CRI

Reply:
> we don't normally include library level changes

I think we should make an exceptional case here to have "Library changes" section, because now the caller of oci.WithPrivileged needs to be as privileged as possible

Reply:
Sure, we aren't limited to this format either. I try to keep this list short and sweet, but in the past we have included more detail. We can further detail the impactful changes as part of this release. This is certainly not the only big change for clients in this release.

Reply:
Please feel free to add a section to the 1.5.0 release notes at any time and we will get into the next beta/rc. I'll try to think about more content we can add there as well.